pkgs/games/gcompris/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, cmake, qtbase, fetchurl, qtdeclarative, qtmultimedia, qttools, qtsensors, qmlbox2d, gettext, qtquickcontrols, qtgraphicaleffects, makeWrapper,
+  gst_all_1
+}:
+stdenv.mkDerivation rec {
+  version = "0.90";
+  name = "gcompris-${version}";
+
+  src = fetchurl {
+    url = "http://gcompris.net/download/qt/src/gcompris-qt-${version}.tar.xz";
+    sha256 = "1i5adxnhig849qxwi3c4v7r84q6agx1zxkd69fh4y7lcmq2qiaza";
+  };
+
+  enableParallelBuilding = true;
```

Comment:
cmake builds in parallel by default.

Reply:
Explicit is better than implicit no? Do you really want me to remove it?

Reply:
Hmm, there are a lot of other attributes that are implicit:) I also think we can let default values to be applied.

Reply:
Done.